source/recurring_payments/index.html.md.erb

```diff
@@ -192,28 +193,27 @@ Your request should look something like this:
 {
   "amount": 1000,
   "description": "Council tax payment for 2022/23",
+  "reference": "CT-payment1",
   "authorisation_mode": "agreement",
   "agreement_id": "cgc1ocvh0pt9fqs0ma67r42l58"
 }
 ```
 
 The API returns a standard create payment response. Successful responses include the `agreement_id` you sent and `authorisation_mode` is `agreement`.
 
-GOV.UK Pay uses a token representing the user’s saved payment details from the agreement to automatically take the payment.
+GOV.UK Pay uses a token representing the user’s saved payment details from the agreement to take the payment.
```

Comment:
is representing the right word here? Would reflecting or 'that captures' work better?

Reply:
Looking at other payment providers' docs, 'representing' seems fairly standard, as in [Stripe](https://stripe.com/docs/api/tokens?lang=go) and [Global Payments](https://www.globalpaymentsintegrated.com/en-us/blog/2020/05/05/payment-tokenization-guide).

There's a conversation within Pay to remove talk of 'tokenisation' anyway because services don't really need to know, so this line might be gone in a few days, anyway!